web/interfaces.js

```diff
@@ -181,7 +180,6 @@ class IPDFTextLayerFactory {
    * @returns {TextLayerBuilder}
    */
   createTextLayerBuilder({
-    textLayerDiv,
```

Comment:
Please remember to add the new `isOffscreenCanvasSupported` parameter below, and also in the JSDocs above.

Reply:
Arff... why isn't `gulp typestest` failing here ??